app/session.go

```diff
@@ -83,7 +85,7 @@ func (a *App) GetSession(token string) (*model.Session, *model.AppError) {
 
 	if session.Id == "" {
 		var nErr error
-		if session, nErr = a.Srv().Store.Session().Get(token); nErr == nil {
+		if session, nErr = a.Srv().Store.Session().Get(sqlstore.WithMaster(context.Background()), token); nErr == nil {
```

Comment:
Nice, you have already solved https://mattermost.atlassian.net/browse/MM-32620 :)